.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["es2015", "react", "stage-0"],
-  "plugins": ["add-module-exports"]
+  "plugins": ["add-module-exports", "transform-flow-strip-types"]
```

Comment:
hm,... depends on your dependencies... if there are no dependencies, it's easier to consume... otherwise we should publish separate libdefs on flow-typed for now